generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.state.ts

```diff
@@ -0,0 +1,32 @@
+import { Routes } from '@angular/router';
+
+import { UserRouteAccessService } from '../../shared';
+import { NavbarComponent } from './navbar.component';
+import { AuthService } from '../../shared';
+
+// @Injectable()
+// export class AuthorizeResolve implements Resolve<any> {
+//
+//   constructor(private authService: AuthService) {}
+//
+//   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot) {
+//       return this.authService.authorize();
+//   }
+//
+// }
+
+export const navbarRoute: Routes = [
```

Comment:
its still weird to have navbar as a state. may be this should be done as app.state or main-state